src/libcore/cmp.rs

```diff
@@ -53,12 +53,42 @@ use option::Option::{self, Some};
 /// symmetrically and transitively: if `T: PartialEq<U>` and `U: PartialEq<V>`
 /// then `U: PartialEq<T>` and `T: PartialEq<V>`.
 ///
+/// ## Derivable
+///
+/// This trait can be used with `#[derive]`. When `derive`d on structs, two
+/// instances are equal if all fields are equal, and non equal if any fields
```

Comment:
Oooh yep. That's what I get for [copy pasting without revising](https://github.com/rust-lang/rust/blob/master/src/libsyntax_ext/deriving/cmp/partial_eq.rs#L27-L28).